front/src/weekview/Weekview.js

```diff
@@ -411,18 +409,19 @@ class Weekview extends Component {
       const paramMonth = urlParamDateSplit[1];
       const paramYear = urlParamDateSplit[0];
 
-      const paramDateCorrect = moment(paramYear + paramMonth + paramDay, 'YYYYMMDD');
-
+      const paramDateCorrect = moment(paramYear + paramMonth + paramDay, 'YYYYMMDD').toDate();
       // Jos viikkonumero ei oo oikee laitetaan current
       if (isNaN(weeknumber)) { // eslint-disable-line
         this.setState({ weekNro: current });
         // Tähän viel että parametriks tulee tän hetkinen viikko
         const now = moment().format();
-        this.props.history.replace(`/weekview/${now}`);
+        this.props.history.replace(`/weekview/${now.substring(0, 10)}`);
       } else {
         // Jos on oikee nii laitetaan url params
         // dayNro pitäs saada parametrien mukaan oikeeks
```

Comment:
Not your fault, but could you fix these comments and others in this file from earlier team please

Reply:
Cleaned/removed comments